Azure/azure-cli
# [Network] `az network application-gateway`: Support OCSP revocation check on client certificate

**Related command**

az network application-gateway create
az network application-gateway update

**Description**

Resolves: https://github.com/Azure/azure-cli/issues/24011
Need to complete `az network application-gateway ssl-profile` in the future.

**Testing Guide**

```
az network application-gateway update -n {} --ssl-profiles [0].client-auth-configuration.verify-client-revocation=OCSP
```

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
